frontend/src/components/MeetingRoom/Toolbar/Toolbar.tsx

```diff
@@ -133,15 +133,19 @@ const Toolbar = ({
   // Displays the right panel buttons - any additional buttons to be displayed that aren't in the center of the toolbar.
   const displayRightPanelButtons = (toolbarButton: ReactElement | false, index: number) =>
     index >= centerButtonLimit && index < rightButtonLimit && toolbarButton;
+  // Array of `false` or right panel button ReactElements to display.
+  const rightPanelButtons = toolbarButtons.map(displayRightPanelButtons);
+  // We display the right panel if we have at least one right panel button to display.
+  const displayRightPanel = rightPanelButtons.reduce((sum, next) => sum + Number(!!next), 0) >= 1;
```

Comment:
great comments! 💪

Reply:
mmm you could use array.some to simplify